py/server/deephaven/table_listener.py

```diff
@@ -509,23 +515,15 @@ def __init__(self, tables: Sequence[Table], listener: Union[Callable[[Dict[Table
         except Exception as e:
             raise DHError(e, "failed to create a merged listener adapter.") from e
 
-    def _process(self, replay_updates: Optional[jpy.JType] = None) -> None:
-        """Process the table updates from the listener recorders.
-
-        Args:
-            replay_updates (Optional[jpy.JType]): the replay updates, only provided during replay.
-        """
-        if replay_updates:
-            self.listener({t: TableUpdate(t, tu) for t, tu in zip(self.tables, j_list_to_list(replay_updates))}, True)
-        else:
-            self.listener({lr.table: lr.table_update() for lr in self.listener_recorders}, False)
+    def _process(self) -> None:
+        """Process the table updates from the listener recorders. """
+        self.listener({lr.table: lr.table_update() for lr in self.listener_recorders}, False)
```

Comment:
```suggestion
        self.listener({lr.table: TableUpdate(lir.table, r.table_update()) for lr in self.listener_recorders}, False)
```
I think the Java value needs to be wrapped here.  If so, the unit tests missed this.

Reply:
They are the Python wrappers and return the TableUpdate wrappers.